nb_conda/nbextension/envmanager.py

```diff
@@ -79,7 +76,7 @@ def get_info(env):
 
     def delete_env(self, env):
         output = self._execute('conda env remove -y -q --json -n', env)
-        output = '\n'.join(output.splitlines()[1:]) # discard 'Fetching package metadata...'
+        output = '\n'.join(output.splitlines()[1:])  # discard 'Fetching package metadata...'
```

Comment:
Yeah, I've been using SublimeLinter-flake8 on save. It's just annoying enough with the warnings to get me to fix them.